platforms/js/TextClip.hx

```diff
@@ -205,6 +205,15 @@ class TextClip extends NativeWidgetClip {
 		style.wordWrap = false;
 		style.wordWrapWidth = 2048.0;
 
+		this.onAdded(function() {
```

Comment:
Looks like we can keep 'disableSprites' only for TextClip. Just subscribe on clip added and keep this subscription until it is removed.